keras_cv/layers/preprocessing/randomly_zoomed_crop.py

```diff
@@ -79,7 +79,7 @@ def __init__(
         )
 
         self._check_class_arguments(height, width, zoom_factor, aspect_ratio_factor)
-
+        self.force_output_dense_images = True
```

Comment:
this seems to be introducing more user cognitive load, is there any other way to reduce it?